src/test/java/nl/dtls/fairdatapoint/utils/ExampleFilesUtils.java

```diff
@@ -92,8 +92,7 @@
     public static final String BASE_URI = "http://localhost/";   
     public static final String TEST_SUB_URI = "http://www.dtls.nl/test"; 
     public static final RDFFormat FILE_FORMAT = RDFFormat.TURTLE;    
-    private static final Logger LOGGER = 
-            LogManager.getLogger(ExampleFilesUtils.class.getName());
+    private static final Logger LOGGER = LoggerFactory.getLogger(ExampleFilesUtils.class.getName());
```

Comment:
We can use the class itself instead of resolving it per name. (also for consistency)